code/modules/uplink/uplink_items.dm

```diff
@@ -1832,6 +1832,15 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	cost = 2
 	restricted_species = list("plasmaman")
 
+/datum/uplink_item/race_restricted/tribal_claw
+	name = "Old Tribal Scroll"
+	desc = "(ONLY USABLE BY LIZARDS) We found this scroll in a abandoned lizard settlement of the Knoises clan. \
+			It teaches you how to use your claws and tails to gain an advantage in combat"
```

Comment:
you could make the lizard only warning as part of the flavour, it doesn't need to be this obtrusive